integration/smoke_tests/form_spec.rb

```diff
@@ -74,9 +74,7 @@ def load(expansion_or_html = {}, &block)
 
     form.send_application_button.click
     attachments = EmailAttachmentExtractor.find(
-      id: generated_name,
-      pdf_filename: '-answers.pdf',
-      user_attachment_filename: 'hello_world.txt'
```

Comment:
We don't need to pass what file on the attachment because we are looking for the email containing the subject "X" or the attachments "Y". So the id is enough 🚀  #lesscomplexity